domain/timeline/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/timeline/TimelineEvent.kt

```diff
@@ -94,6 +94,9 @@ enum class TimelineEventType {
   STEP_STARTED,
   STEP_COMPLETED,
 
+  // Action Plan Review events
+  ACTION_PLAN_REVIEW_COMPLETED,
+
   // Conversation/Notes events
   CONVERSATION_CREATED,
```

Comment:
Nothing to do with this PR... Should we remove the conversation stuff? or create a ticket to do it?